components/locid_transform/src/fallback/algorithms.rs

```diff
@@ -243,7 +244,7 @@ mod tests {
             requires_data: false,
             extension_key: None,
             fallback_supplement: None,
-            expected_language_chain: &["en-US-u-sd-usca", "en-US", "en"],
+            expected_language_chain: &["en-US-u-sd-usca", "en-US", "en-u-sd-usca", "en"],
```

Comment:
Nit: I would prefer if you put the locid_transform changes into its own PR because it is a standalone behavior change that should be independently trackable. If this is too hard (don't know why it would be), then change this PR title to mention it.

Reply:
#3594